modules/apps/object/object-web/src/main/resources/META-INF/resources/type-declarations/data-engine-js-components-web.d.ts

```diff
@@ -13,13 +13,11 @@
  */
 
 declare module 'data-engine-js-components-web' {
-	function FieldFeedback({
-		errorMessage,
-		helpMessage,
-		warningMessage,
-	}: {
-		errorMessage?: string;
-		helpMessage?: string;
-		warningMessage?: string;
-	}): JSX.Element;
+	export * from 'data-engine-js-components-web/src/main/resources/META-INF/resources/js/index';
+
+	function useFeatureFlag(): {
+		[key in Flags]: boolean;
+	};
 }
+
+type Flags = '';
```

Comment:
I'm not sure, but maybe you'll need to regenerate this file because now we're using the global Liferay object and this won't be necessary.
see https://github.com/liferay-frontend/liferay-portal/pull/2260

Reply:
Thanks for letting me know @alinedoleron! I did not know that! :smile: